### DISCUSSION TITLE
Elastic Search Hosting Recommendations

### DISCUSSION THREAD

**Discussion by mariojsnunes** [OrchardCMS/OrchardCore#9439] [Q&A]
So with the new Elastic Search module comming up (https://github.com/OrchardCMS/OrchardCore/pull/9108), what is your hosting choice/recommendation for a multi-tenant app?

**Comment by Skrypt** [comment]
To be honest I don't know yet. I know that ElasticSearch has free trial hosting. You should be able to also use any Docker container if you want to manage it by yourself.

**Comment by mariojsnunes** [comment]
@Sen-Gupta do you have any recommendation for this?
